### PR TITLE
Create script file: show_error.php

### PR DESCRIPTION
**Pull request by Jarne-Verlinden**

### CODE REVIEW

**Review by iorWout — COMMENTED** [review]
Reviewed code

File: scripts/show_error.php [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?php
+function showError($err_type, $err_message)
+{
+    echo "Error: Type: " . $err_type . ", message: " . $err_message . ", err-id: " . uniqid();
```
Comment (iorWout) [inline scripts/show_error.php:4]:
```suggestion
    echo "Error: Type: " . $err_type . ", message: " . $err_message . ", generated id: " . uniqid();
```

**Review by iorWout — CHANGES_REQUESTED** [review]
Reviewed code

**Review by Jarne-Verlinden — COMMENTED** [review]
Changed line code

**Pull request merged**